src/api/index.ts

```diff
@@ -68,7 +68,7 @@ const api = (networkName: NetworkName, version: ApiVersion = "v1") => {
       }) {
         const res = await apiClient.get<Notification[]>(`/notices`, {
           params,
-          baseURL: getBaseUrl("mainnet", version),
+          baseURL: getBaseUrl("xpla", version),
```

Comment:
this endpoint is dezswap-api, which should not be affected from changed network name